test/data/network/network-1.tsv.data

```diff
@@ -0,0 +1,5 @@
+regulator target attr1 attr2 attr3
```

Comment:
What would happen if we don't add this? I thought it would somehow success as well? There isn't a qunit test that checks the rendered network.